docs/local_resource.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Run Local and/or Occasional Workflows with Local Resource
+layout: docs
+---
+A Local Resource works like any other resource in your sidebar (which are most
+often some combination of Docker Build instructions and K8s Yaml)--encompasses
+a unit of work, and execute in response to file changes, or
+[manually](https://docs.tilt.dev/manual_update_control.html). For your resource
+`MyGreatService`, when one of its file dependencies changes, the work it executes
+is to build a Docker image and deploy some k8s yaml; for a Local Resource, it's
+to execute an arbitrary command on your local filesystem.
+
+# Run commands locally on file change
+
+Tilt is designed to make it easy to build images, and to sync files/run commands
+in the cloud. For some kinds of development, however, you might want to harness
+Tilt's responsiveness to file changes, but run commands locally instead. These
```

Comment:
"These cases often fall into the general categories of" makes this list sound more exhaustive than I think is intended. Maybe "Some common use cases we've seen this address"?